base.yml

```diff
@@ -1,245 +1,1084 @@
+---
 AllCops:
-  NewCops: enable
```

Comment:
I believe removing this will make Rubocop warn us about new cops that we haven’t yet configured.